src/Writers/HTMLWriter.jl

```diff
@@ -1183,4 +1217,172 @@ fixlinks!(ctx, navnode, md) = nothing
 # TODO: do some regex-magic in raw HTML blocks? Currently ignored.
 #fixlinks!(ctx, navnode, md::Documents.RawHTML) = ...
 
+# Algolia search
+# ------------------------------------------------------------------------------
+
+# TODO create better settings, now all defaults
+"""Customized Algolia settings for optimal results
```

Comment:
```suggestion
"""
Customized Algolia settings for optimal results
```

Minor style bikeshed, but the docstrings in Documenter usually start on a new line. Ditto for some of the others.